projects/jobhunter.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
This file also seems extra for the dark mode PR because we already have three projects to test on. This project  might just lead to a merge conflict later.